### PR TITLE
Remove marketing content and dummy features/buttons from Portal

### PR DESCRIPTION
**Pull request by griffri**
Resolves https://github.com/dlcs/private-protagonist/issues/121

Changes:
- Marketing-related content has been removed
- Non-functional links, buttons and inputs have been removed
- Several pages with identical layouts (e.g the main page, login-related pages) now share a common layout file to help maintain consistency

image
image

### CODE REVIEW

File: src/protagonist/Portal/Pages/Account/SignupFromLink.cshtml [hunk 1]
```diff
@@ -66,17 +49,20 @@
                     <label asp-for="Input.Slug">URL name</label>
                     <input asp-for="Input.Slug" class="form-control" aria-describedby="slugHelp" placeholder="my-org">
                     <span asp-validation-for="Input.Slug" class="text-danger"></span>
-                    <small id="slugHelp" class="form-text text-muted">Your IIIF Image URLs will include this as part of the address.
-                        For example, dlcs.io/iiif-img/<b>my-org</b>/3/my-deep-doom-image.</small>
+                    <small id="slugHelp" class="form-text text-muted">
+                        Your IIIF Image URLs will include this as part of the address.
+                        For example, dlcs.io/iiif-img/<b>my-org</b>/3/my-deep-doom-image.
```
Comment (JackLewis-digirati) [inline src/protagonist/Portal/Pages/Account/SignupFromLink.cshtml:54]: I know this was here before but is this supposed to be `my-deep-zoom-image`

File: src/protagonist/Portal/Pages/Index.cshtml [hunk 2]
```diff
@@ -1,205 +1,124 @@
...
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="list"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Queue/Index">Queue</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+                -->
```
Comment (JackLewis-digirati) [inline src/protagonist/Portal/Pages/Index.cshtml:52]: commented out code

File: src/protagonist/Portal/Pages/Index.cshtml [hunk 3]
```diff
@@ -1,205 +1,124 @@
...
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="search"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Queries/Index">Named queries</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+                -->
```
Comment (JackLewis-digirati) [inline src/protagonist/Portal/Pages/Index.cshtml:102]: commented out code

File: src/protagonist/Portal/Pages/Shared/_ControlPanelLayout.cshtml [hunk 4]
```diff
@@ -1,123 +1,116 @@
...
+          <li class="nav-item">
+            <navlink href="/origins" icon="globe">Origins</navlink>
+          </li>
+          <li class="nav-item">
+            <navlink href="/auth" icon="lock">Access control</navlink>
+          </li>
+          <li class="nav-item">
+            <navlink href="/queries" icon="search">Named queries</navlink>
+          </li>
+          -->
```
Comment (JackLewis-digirati) [inline src/protagonist/Portal/Pages/Shared/_ControlPanelLayout.cshtml:70]: commented out code

**Review by JackLewis-digirati — APPROVED** [review]

**Review by tomcrane — APPROVED** [review]

File: src/protagonist/Portal/Pages/Shared/_ControlPanelLayout.cshtml (hunk 4 above)
Comment (griffri) [reply src/protagonist/Portal/Pages/Shared/_ControlPanelLayout.cshtml]: Removed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)

File: src/protagonist/Portal/Pages/Index.cshtml (hunk 3 above)
Comment (griffri) [reply src/protagonist/Portal/Pages/Index.cshtml]: Removed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)

File: src/protagonist/Portal/Pages/Index.cshtml (hunk 2 above)
Comment (griffri) [reply src/protagonist/Portal/Pages/Index.cshtml]: Removed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)

File: src/protagonist/Portal/Pages/Account/SignupFromLink.cshtml (hunk 1 above)
Comment (griffri) [reply src/protagonist/Portal/Pages/Account/SignupFromLink.cshtml]: Turns out it was - fixed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)

**Pull request merged**